Explorer++/ShellBrowser/SortManager.cpp

```diff
@@ -376,14 +386,28 @@ int CALLBACK CShellBrowser::SortByName(int InternalIndex1,int InternalIndex2) co
 		{
 			/* If the items been compared are both drives,
 			sort by drive letter, rather than display name. */
-			return StrCmpLogicalW(FullFileName1,FullFileName2);

```

Comment:
I don't think the change would be required here, since the strings being compared are drive letters, and so shouldn't contain any numbers.